tst/regression/linear_modes.gold

```diff
@@ -1,1232 +1,1232 @@
-1.000005628528739532e+00
```

Comment:
I guess if it converges, it's fine, but it would be good to understand where the diff is coming from

Reply:
Oh you know I bet this is because we're now correctly setting `gm1` in initialization in the linear modes pgen:
```c++
 v(igm1, k, j, i) = eos.BulkModulusFromDensityTemperature(
                               v(irho, k, j, i), v(itmp, k, j, i), eos_lambda) /
                           v(iprs, k, j, i);
                           ```

Reply:
oooh yeah ok.